lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -312,36 +318,41 @@ defmodule Mix.Tasks.Xref do
   ## Callers
 
   defp do_callers(filter) do
```

Comment:
Since we are changing around here, should we call this `print_callers` or `format_callers`? We are getting rid of the `do_` prefix.